docs/src/assets/scss/docs.scss

```diff
@@ -146,10 +146,7 @@ pre[class*="language-"] {
     right: 0.5rem;
     offset-block-end: 0.5rem;
     offset-inline-end: 0.5rem;
-
-    @media all and (max-width: 768px) {
-        display: none;
-    }
```

Comment:
This would be an accessibility issue. Users on less-wide screens, such as mobile phones and zoomed-in desktop displays, still would want to be able to access the _Open in Playground_ button.

Reply:
For this, I'll have to ask @nzakas what the original intention was.

The Playground is quite good on small screens, but this button is very likely to cover the code:

<details>

![Screenshot_2023-07-08-16-45-43-843_com android chrome](https://github.com/eslint/eslint/assets/44349756/e0ddf050-d3df-4fee-9378-34e92269de18)

</details>

Though that can happen on wider screens too, depending on the text. Perhaps the button should be below the code block?

Reply:
I like the idea of moving the button below in mobile screens, yeah. Will wait to implement until directed to 👍

Reply:
Yeah, the intent was that smaller screens (mobile) would probably not want to use the playground as it's a bit clunky without a keyboard.

However, I'm not opposed to moving the button underneath the code block. That seems like a nice compromise to keep the functionality available without  disrupting the UI too much.

Reply:
Super, that's two votes in favor - I had a 💡 moment and moved the button to be half overlapping, half under the code block. That way it takes up much less vertical space and doesn't overlap lines of code.